addons/purchase_requisition_stock/models/purchase.py

```diff
@@ -12,3 +12,19 @@ def _onchange_requisition_id(self):
         super(PurchaseOrder, self)._onchange_requisition_id()
         if self.requisition_id:
             self.picking_type_id = self.requisition_id.picking_type_id.id
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'purchase.order.line'
+
+    @api.model
+    def _prepare_purchase_order_line_from_procurement(self, product_id, product_qty, product_uom, company_id, values, po):
+        res = super()._prepare_purchase_order_line_from_procurement(product_id, product_qty, product_uom, company_id, values, po)
+        if po.requisition_id:
+            requisition_line = po.requisition_id.line_ids.filtered(lambda line: line.product_id == product_id)
+            if requisition_line.product_description_variants:
+                if res['name']:
+                    res['name'] += '\n' + requisition_line.product_description_variants
+                else:
+                    res['name'] = requisition_line.product_description_variants
```

Comment:
Is it really possible that `res['name']` isn't defined?
Also, I wonder if it should be added to the `res['product_description_variants']` as well. :confused:

Reply:
You are correct regarding `res['name']`. But I didn't know before about `product_description_variants` field in purchase order line 🤔 . As I understand, it is not shown in any view, what is it used for then?

Reply:
Had to dig a bit for that one, it was previously used in the moves generation from the PO lines, but that doesn't seem to be the case anymore. You're right, I don't see any uses from the one defined in the PO line, the only time it's used it's from a procurement's value, which is already created at this point anyway.